src/com/amazon/ion/util/Equivalence.java

```diff
@@ -529,7 +619,24 @@ public static boolean ionEquals(final IonValue v1,
     public static boolean ionEqualsByContent(final IonValue v1,
                                              final IonValue v2)
     {
-        return ionEqualsImpl(v1, v2, false);
+        return ionEqualsImpl(v1, v2, NON_STRICT_CONFIGURATION);
+    }
+
+    /**
+     * Checks for data equivalence over two Ion values using this Equivalence's
+     * configuration
+     *
+     * @see Builder
+     *
+     * @param v1
```

Comment:
Breaking the `@param`s across extra lines is unorthodox. Did your IDE do this automatically?

Reply:
I agree, but I did it to be consistent with the style of the javadocs on `ionEquals` and `ionEqualsByContent`.